app/src/main/java/com/example/playlistmaker/SearchActivity.kt

```diff
@@ -97,11 +108,13 @@ class SearchActivity : AppCompatActivity() {
         historyVisible()
 
         fun sendToServer() {
-            trackRecyclerView.visibility = View.VISIBLE
-            nothingFoundCase.visibility = View.GONE
-            errorNoConnection.visibility = View.GONE
-            searchHistoryLayout?.visibility = View.GONE
             if (searchQueryText?.text?.isNotEmpty()!!) {
```

Comment:
🍏 Double bang (!!) важно использовать аккуратно, он может привести к неожиданному падению на ровном месте.

```suggestion
            if (searchQueryText?.text?.isNotEmpty() == true) {
```